Weekly-Store-Sales.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Weekly Store Market Sales By Region</title><!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <style>
+    .header {
+      text-align: center;font-style: italic; color:green;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <div class="header">
+      <h1>Weekly Store Sales by Market</h1>
+      <u>Comparison of Medium to Large Markets.</u>
```

Comment:
⚠️ The U tag is deprecated, we shouldn't use it. This would be better as a P (paragraph), that we could style like this: `.header p { text-decoration: underline; }

Also note that underline typically signals link, so we have to be careful of using it on the web.

Would be great to give a source for this data and more information about what it is.